### PR TITLE
chore: update getItemTag to react component

### PR DESCRIPTION
**Pull request by li-jia-nan**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [x] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English | docs: update docs |
| 🇨🇳 Chinese | docs: update docs |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

File: README-zh_CN.md [hunk 1]
```diff
@@ -63,7 +63,7 @@
 - 📦 开箱即用的高质量 React 组件。
 - 🛡 使用 TypeScript 开发，提供完整的类型定义文件。
 - ⚙️ 应用开发框架和设计工具配套。
-- 🌍 数十个国际化语言支持。
+- 🌍 数六十个国际化语言支持。
```
Comment (yoyo837) [inline README-zh_CN.md:66]: 不通顺

File: docs/react/introduce.zh-CN.md [hunk 2]
```diff
@@ -19,7 +19,7 @@ title: Ant Design of React
 - 📦 开箱即用的高质量 React 组件。
 - 🛡 使用 TypeScript 开发，提供完整的类型定义文件。
 - ⚙️ 全链路开发和设计工具体系。
-- 🌍 数十个国际化语言支持。
+- 🌍 数六十个国际化语言支持。
```
[the remark this answers is not in the document]
Comment (li-jia-nan) [reply docs/react/introduce.zh-CN.md:22]: 我想到了，改成 “数百个” 好了

File: README-zh_CN.md (hunk 1 above)
Comment (li-jia-nan) [reply README-zh_CN.md:66]: 我想到了，改成 “数百个” 好了

File: docs/react/introduce.zh-CN.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (yoyo837) [reply docs/react/introduce.zh-CN.md]: 数百个？实际不足百个吧？

[the remark this answers is not in the document]
Comment (li-jia-nan) [reply docs/react/introduce.zh-CN.md]: 实际六七十个，也差不多了，写个数十种也太少了

[the remark this answers is not in the document]
Comment (li-jia-nan) [reply docs/react/introduce.zh-CN.md]: 原则上我们可以四舍五入

[the remark this answers is not in the document]
Comment (yoyo837) [reply docs/react/introduce.zh-CN.md]: 数十种就是几十个的意思，写数百个是虚报。

[the remark this answers is not in the document]
Comment (kiner-tang) [reply docs/react/introduce.zh-CN.md]: 感觉还是原先的数十个更准确一点，而且也符合实际情况

[the remark this answers is not in the document]
Comment (yoyo837) [reply docs/react/introduce.zh-CN.md]: > 原则上我们可以四舍五入

个人认为这不是四舍五入，是虚报

[the remark this answers is not in the document]
Comment (yoyo837) [reply docs/react/introduce.zh-CN.md]: 中英文都用“70 +”，这样如何？

[the remark this answers is not in the document]
Comment (li-jia-nan) [reply docs/react/introduce.zh-CN.md]: @afc163 老板看看有什么好的意见没？

[the remark this answers is not in the document]
Comment (MadCcc) [reply docs/react/introduce.zh-CN.md]: 数个：1 ～ 9
数十个：10 ～ 99
十数个：10 ～ 19

**Comment by li-jia-nan** [comment]
这个分支正好还在，我拿来改点其它东西，把之前大家无关的评论删掉了哈

**Review by afc163 — APPROVED** [review]

**Pull request merged**